tests/FSharp.Compiler.Service.Tests/FSharp.CompilerService.SurfaceArea.netstandard.expected

```diff
@@ -7157,9 +7157,11 @@ FSharp.Compiler.Syntax.SynField: Boolean get_isMutable()
 FSharp.Compiler.Syntax.SynField: Boolean get_isStatic()
 FSharp.Compiler.Syntax.SynField: Boolean isMutable
 FSharp.Compiler.Syntax.SynField: Boolean isStatic
-FSharp.Compiler.Syntax.SynField: FSharp.Compiler.Syntax.SynField NewSynField(Microsoft.FSharp.Collections.FSharpList`1[FSharp.Compiler.Syntax.SynAttributeList], Boolean, Microsoft.FSharp.Core.FSharpOption`1[FSharp.Compiler.Syntax.Ident], FSharp.Compiler.Syntax.SynType, Boolean, FSharp.Compiler.Xml.PreXmlDoc, Microsoft.FSharp.Core.FSharpOption`1[FSharp.Compiler.Syntax.SynAccess], FSharp.Compiler.Text.Range)
```

Comment:
No, not at all 😸
I pretty much break this on a weekly basis: https://github.com/dotnet/fsharp/commits/0b7ea617d184cc8862016782075cb4911c6c90f7/tests/FSharp.Compiler.Service.Tests/FSharp.CompilerService.SurfaceArea.netstandard.expected

That has never been a concern so far.

Reply:
I think there a few rules/practices helping us with FCS relying tooling:

- Pattern matching must be prepared for new cases coming up with new versions (so update with the change, or have a discard match clause in the end)

- Expanding tuples carrying data will not break as long as pattern matches use named field instead of positional. And in case of positional, it is an easy update.